web/package.json

```diff
@@ -64,7 +71,7 @@
     "protractor": "5.4.0",
     "ts-mockito": "2.3.0",
     "ts-node": "4.1.0",
-    "tslint": "5.9.1",
+    "tslint": "^5.9.1",
```

Comment:
still necessary, even after removal?

Reply:
It shouldn't be, but my console required it...
Maybe it's because of the 'ng lint' settings which I didn't change yet.

Reply:
But pls try it without this dependency - maybe it's only a local problem of mine!

Reply:
I thought so, but I had problems when I linted with my cmd.
After the rebase it works without this dependecy so I think it's ok to remove it!

Reply:
It's necessary for the eslint-tslint plugin